synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -445,6 +445,13 @@ export class SynopsysToolsParameter {
       const parsedUrl = url.parse(collectionUri);
       azureInstanceUrl = `${parsedUrl.protocol}//${parsedUrl.host}`;
       azureOrganization = parsedUrl.pathname?.split("/")[1] || "";
+      if (
+        parsedUrl.host &&
+        !azureOrganization &&
+        parsedUrl.host.indexOf("visualstudio.com") !== -1
+      ) {
+        azureOrganization = parsedUrl.host.split(".")[0];
```

Comment:
add only if parsedUrl.host.split(".")[0] exist

Reply:
this would be sufficient to set the orgname